src/lib.rs

```diff
@@ -1908,6 +1897,15 @@ pub(crate) mod test {
                 Complex::new(1.65826, -0.33502),
                 1e-5
             ));
+            let z = Complex::new(0.0, 0.0);
+            assert!(close(z.powc(b), z));
+            assert!(z.powc(Complex64::new(0., 0.)).is_nan());
```

Comment:
I think this result should be one -- in fact the standard `f64::powf(x, 0.0) == 1.0` for *any* base, even NaN! I'm going to fix both `powf` and `powc` here...

Reply:
That's not mathematically correct though, 0^0 is [indeterminate](https://en.wikipedia.org/wiki/Indeterminate_form).

Reply:
See the prior discussions in <https://github.com/rust-num/num-traits/pull/78> and <https://github.com/rust-num/num-traits/pull/79>. It's a pragmatic choice, especially to be consistent with `std`.

Reply:
That wikipedia article also says:
> Another example is the expression 0^0. Whether this expression is left undefined, or is defined to equal 1, depends on the field of application and may vary between authors. For more, see the article [Zero to the power of zero](https://en.wikipedia.org/wiki/Zero_to_the_power_of_zero).